Client/Nimbus/OnboardingFeatureLayer/NimbusOnboardingFeatureLayer.swift

```diff
@@ -16,7 +16,7 @@ class NimbusOnboardingFeatureLayer: NimbusOnboardingFeatureLayerProtocol {
         return OnboardingViewModel(
             cards: getOrderedOnboardingCards(from: framework.cards,
                                              using: framework.cardOrdering),
-            dismissable: framework.dismissable)
+            isDismissable: framework.dismissable)
```

Comment:
noticed this when I got a compiler error building main, looks like this was as easy renaming the argument so I did it :)